tools/clang_format.py

```diff
@@ -23,6 +21,9 @@
 CLANG_FORMAT_WHITELIST = ["torch/csrc/jit/", "test/cpp/jit/"]
 
 CPP_FILE_REGEX = re.compile("^.*\\.(h|cpp|cc|c|hpp)$")
+CPP_FILE_REGEX = re.compile(".*\\.(h|cpp|cc|c|hpp)$")
```

Comment:
Can we put this in a separate patch please?

Reply:
oops this got included in somehow, will remove